.travis.yml

```diff
@@ -3,19 +3,9 @@ sudo: false
 dist: trusty
 
 cache:
-    directories:
-        - vendor
-        - $HOME/.composer/cache
-
-php:
-  - 7.1
-  - 7.2
-  - nightly
```

Comment:
Please avoid changing this if possible.

Reply:
I need to check if I can re-introduce these, without the matrix magically creating new jobs. At first glance this may work. What's the point, though?